libsolidity/formal/Why3Translator.cpp

```diff
@@ -428,7 +428,7 @@ bool Why3Translator::visit(BinaryOperation const& _binaryOperation)
 	Type const& commonType = *_binaryOperation.annotation().commonType;
 	Token::Value const c_op = _binaryOperation.getOperator();
 
-	if (commonType.category() == Type::Category::IntegerConstant)
+	if (commonType.category() == Type::Category::NumberConstant)
```

Comment:
Check that denominator is 1, otherwise call `error(_identifier, "Fractional numbers not supported.")`.